### DISCUSSION TITLE
Copy to clipboard does not work

### DISCUSSION THREAD

**Discussion by lakshaa** [globaleaks/globaleaks-whistleblowing-software#3732] [Q&A] [answered]
The option "copy to clipboard" does not work, either for the TOR link or for the final receipt number. For the final receipt number, we cannot even highlight the number to manual copy it (CTRL + C or Right Click). We have to note down each number manually. Help please, thank you.

![onion copy](https://github.com/globaleaks/GlobaLeaks/assets/22730590/86920f91-59d0-45d9-aa96-0eaa0b31cc78)

**Comment by evilaliv3** [comment]
Thank you for reporting this.

I suspect It could be related to the type/version of your browser.

Which browser are you using?
Are you accessing via Tor or HTTPS?

**Reply by lakshaa** [reply]
Thank you for replying. I have tried using Chrome, Firefox and Edge, does not work for any of these. Accessing via HTTP for now.

**Comment by evilaliv3** [comment, marked as answer]
Okay, the problem is that you are accessing via HTTP.
This functionality is available only over HTTPS or Tor.

In the up to date version of GlobaLeaks HTTPS is always implemented so you wont incurr in any issue.
